connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -315,9 +319,34 @@ public void testReplication() throws InterruptedException {
             backup.kafka().consume(NUM_RECORDS_PRODUCED, 2 * RECORD_TRANSFER_DURATION_MS, "primary.test-topic-2").count());
     }
 
+    private void waitForConsumerGroupOffsetSync(Consumer<byte[], byte[]> consumer, List<String> topics)
+            throws InterruptedException {
+        Admin backupClient = backup.kafka().createAdminClient();
+        List<TopicPartition> tps = new ArrayList<>(NUM_PARTITIONS * topics.size());
+        IntStream.range(0, NUM_PARTITIONS).forEach(
+            partitionInd -> {
+                for (String topic: topics) {
+                    tps.add(new TopicPartition(topic, partitionInd));
+                }
+            }
+        );
+        long expectedTotalOffsets = NUM_RECORDS_PRODUCED * topics.size();
+
+        waitForCondition(() -> {
+            Map<TopicPartition, OffsetAndMetadata> consumerGroupOffsets =
+                backupClient.listConsumerGroupOffsets("consumer-group-1").partitionsToOffsetAndMetadata().get();
```

Comment:
could we consider to pass in the consumer group name as a input variable of `waitForConsumerGroupOffsetSync`, so that `waitForConsumerGroupOffsetSync` looks more generic?

Reply:
Good suggestion. Thanks.